app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -140,3 +217,53 @@ $image-overlay-form-width: 350px;
     }
   }
 }
+
+.picture-overlay-navigation {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  width: 100%;
+
+  .previous-picture,
+  .next-picture {
+    position: absolute;
+    top: 0;
+    width: 64px;
+    height: 128px;
+    line-height: 120px;
+    margin-top: -64px;
+    cursor: pointer;
+    text-align: center;
+    text-decoration: none;
+    @include transition(background-color $image-overlay-transition-duration linear);
+
+    &:hover {
+      background-color: rgba(0,0,0, 0.3);
+
+      .icon-angle-left,
+      .icon-angle-right {
+        color: $white;
+        text-shadow: none;
+      }
+    }
+  }
+
+  .icon-angle-left,
+  .icon-angle-right {
+    font-size: 48px;
+    color: $medium-gray;
+    text-shadow: 0 0 $default-padding $text-color;
+    @include transition(all $image-overlay-transition-duration linear);
+  }
+
+  .previous-picture {
+    left: 0;
+    @include border-right-radius($default-border-radius);
+  }
+
+  .next-picture {
+    right: $image-overlay-form-width + $picture-overlay-handle-width;
+    @include border-left-radius($default-border-radius);
```

Comment:
Expected item on line 266 to appear before line 265. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets